tests/plugins/inlinetoolbar/positioning.js

```diff
@@ -0,0 +1,82 @@
+/* bender-tags: inlinetoolbar */
+/* bender-ckeditor-plugins: toolbar,link,inlinetoolbar */
+
+/*
+divarea test are failing, its upstream issue from balloonpanel. issue #1064
+*/
+( function() {
+	'use strict';
+	var createContent = function() {
+		var rawData = '';
+		for ( var i = 0; i < 80; i++ ) {
+			rawData += '<p>This is the editor data.</p>';
+			if ( i === 40 ) rawData +=  '<p>foo bar baz<span id="marker">bom</span></p>';
+		}
+		return rawData;
+	};
+	document.getElementById( 'editor1' ).innerHTML = createContent();
+	document.getElementById( 'divarea' ).innerHTML = createContent();
+
+	bender.editors = {
+		editor1: {
+			name: 'editor1',
+			creator: 'replace',
+			config: {
+				extraAllowedContent: 'span[id]',
+				height: 200,
+				width: 600
+			}
+		},
+		divarea: {
+			name: 'divarea',
+			creator: 'replace',
+			config: {
+				extraPlugins: 'divarea',
+				extraAllowedContent: 'span[id]',
+				height: 200,
+				width: 600
+			}
+		}
+	};
+
+	var tests = {
+
+		'test divaera - out of view - bottom center': function( editor ) {
+			var inlineToolbar = new CKEDITOR.ui.inlineToolbarView( editor, {
```

Comment:
Code style: subsequent `var`s should be declared/initialized in a single `var`, like so:

```js
var inlineToolbar = new CKEDITOR.ui.inlineToolbarView( editor, {
		width: 100,
		height: 200
	} ),
	markerElement = editor.editable().findOne( '#marker' ),
	frame = editor.editable().isInline() ? editor.editable().getClientRect() : editor.window.getFrame().getClientRect(),
	elementFrame = markerElement.getClientRect();
```

Also `inlineToobarRect` should be joined in it.